### PR TITLE
fix #48

### LINKED ISSUE #48: Improve performance of plotting polygons with geopandas
See the issue reported at geopandas: https://github.com/geopandas/geopandas/issues/1146. There is a slow-down with the latest geopandas release. But by looking at that, I noticed that with only a small edit of the `convert_geoDataFrame_to_patches` function, you can make this much faster (regardless of the geopandas version)

### PR DESCRIPTION
**Pull request by lamourj**
Thanks to @jorisvandenbossche who brought the solution in geopandas/geopandas#1146.

### CODE REVIEW

File: pandas_bokeh/geoplot.py [hunk 1]
```diff
@@ -143,11 +143,13 @@ def extract(row, geometry):
...
         if geometry.type == "Polygon":
-            gdf_new = gdf_new.append(extract(row, geometry), ignore_index=True)
+            gdf_new.append(extract(row, geometry))
 
         if geometry.type == "MultiPolygon":
             for polygon in geometry:
-                gdf_new = gdf_new.append(extract(row, polygon), ignore_index=True)
+                gdf_new.append(extract(row, polygon))
+
+    gdf_new = gpd.GeoDataFrame(gdf_new)
```
Comment (jorisvandenbossche) [inline pandas_bokeh/geoplot.py]: This actually also does not need to be a GeoDataFrame, as we are dropping the geometry column just after. Making this a `pd.DataFrame` will avoid another validation of the geometries and a further (although minor) speed-up

**Comment by PatrikHlobil** [comment]
@lamourj : Many Thanks for your contribution. As @jorisvandenbossche mentioned, please change the `gdf_new = gpd.GeoDataFrame(gdf_new)` to a pd.DataFrame and then I will merge the request directly ;).

Best, 

Patrik

**Comment by lamourj** [comment]
> @lamourj : Many Thanks for your contribution. As @jorisvandenbossche mentioned, please change the `gdf_new = gpd.GeoDataFrame(gdf_new)` to a pd.DataFrame and then I will merge the request directly ;).
> 
> Best,
> 
> Patrik

Done :)

**Comment by PatrikHlobil** [comment]
HI @lamourj , 

already merged your branch. Thanks for contributing ;). 

Is there anything you are missing in Pandas-Bokeh or what you would improve?

Best, Patrik

**Comment by lamourj** [comment]
Thanks!
I don't have enough experience with Pandas-Bokeh for now to give you innovative ideas, but I've been quite satisfied with it so far, thanks!

**Pull request merged**